packages/code/src/Code.tsx

```diff
@@ -236,6 +236,12 @@ interface CodeProps extends SyntaxProps {
    * default: `true`
    */
   copyable?: boolean;
+
+  /**
+   * Callback fired when Code is copied
+   *
+   */
+  onCopy: React.MouseEventHandler;
```

Comment:
Couple things:
- I'm inclined to think that, at least this time around, we shouldn't pass anything to this callback (e.g. this is only a function, not a mouse event handler). `onCopy` is a real thing in HTML, and we may want to expand this handler to behave the same in the future.
- This needs to be an optional prop 😄

Reply:
Good points, just updated.